### PR TITLE
Bookstore Styling

### PR DESCRIPTION
**Pull request by butlermuwo**
## Bookstore Styling

In this pull request I did the following:
- I styled the bookstore according to the design on Zeplin

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
+          </li>
+          <li>
+            <Link
+              className="links"
+              activeClassName="activelinks"
+              key="2"
+              to="/categories"
+            >
+              Categories
+            </Link>
```
Comment (moise10r) [inline src/components/Navbar.js:30]: - [ ] Currently, the active page should be indicated with a bold font-weight as on [Zeplin](https://zeplin.io/) design


Screenshot

File: src/components/Bookcard.js [hunk 2]
```diff
@@ -1,41 +1,68 @@
...
             </div>
-          ))
-        }
+          </div>
+          <div className="current-chapter d-flex">
+            <span className="current-chapter-span">Current Chapter</span>
+            <span className="current-chapter-value">
+              Chapter
+              {' '}
+              {}
```
Comment (moise10r) [inline src/components/Bookcard.js]: - [ ] Kindly add this horizontal line between to separate the `completion-detail` with the `current-chapter`

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi @butlermuwo,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
 👍  Good job!

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ]  Your README looks great 👍🏻  but I think you can improve to make it more professional. Kindly add a meaningful project description and also update the screenshot of your project. please make this.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Bookcard.js [hunk 3]
```diff
@@ -1,41 +1,68 @@
...
+            <span className="genre">{}</span>
+            <h3 className="title">{title}</h3>
+            <span className="author">{category}</span>
+            <ul className="book-management d-flex">
+              <li><button type="button">Comments</button></li>
+              <li>
+                {' '}
+                <button type="button" onClick={() => handleRemove(itemId)}>Delete</button>
+              </li>
+              <li><button type="button">Edit</button></li>
```
Comment (moise10r) [inline src/components/Bookcard.js]: - [ ] Kindly remove the `background-color ` you added to the `Comment` `Delete` and `Edit` buttons. As required the styling should EXACTLY be the same as the one provided on [Zeplin](https://zeplin.io/)

|---Figma----|---------Yours--------|
|---------------|-------------------------|
go|bot

Comment (butlermuwo) [reply src/components/Bookcard.js]: Thanks done!

File: src/components/Bookcard.js (hunk 2 above)
Comment (butlermuwo) [reply src/components/Bookcard.js]: Thank you! I have implemented.

File: src/components/Navbar.js (hunk 1 above)
Comment (butlermuwo) [reply src/components/Navbar.js:30]: Cheers! Done.

**Review by moise10r — APPROVED** [review]
### Hi @butlermuwo, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL ]  As requested by the previous reviewer you can improve your README to make it more professional by adding a meaningful project description and also updating the screenshot of your project. please make this.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**